client/components/Message/Actions/index.tsx

```diff
@@ -18,6 +18,9 @@ type ActionOptions = {
 export const Action: FC<ActionOptions> = ({ id, icon, i18nLabel, label, mid, runAction }) => {
 	const t = useTranslation();
 
+	// Legacy icon
+	icon = icon && icon.replace(/videocam/, 'video');
```

Comment:
I particularly don't endorse parameter reassignment (mutating the value of a function parameter) despite being allowed by our linter configuration. For instance, it's harder to notice in larger functions that a parameter value was changed, so some code blocks can still handle it as its value is the same from the function call.

To make the thinking behind this line more explicit, what about adding a function?

```tsx
const resolveLegacyIcon = (legacyIcon: string): string => {
  if (legacyIcon === 'icon-videocam') {
     return 'video';
  }
  
  return icon.replace(/^icon-/, '');
};

const resolvedIcon = resolveLegacyIcon(icon);
```

Also, notice that regular expressions used are not tied to *where* they should check the `icon` string. An accidental value like `'icon-videocam-foo'` would be unnecessarily changed to `'icon-video-foo'`.

Reply:
I like your suggestion, I will follow. by that in a new commit. Thanks!